packages/usa-alert/src/content/test-usa-alert-content.json

```diff
@@ -0,0 +1,102 @@
+{
+  "alerts": [
+    {
+      "header": "Info alert",
+      "title": "Info alert",
+      "modifier": "usa-alert--info",
+      "text": "Lorem ipsum dolor sit amet sed do eiusmod."
```

Comment:
**polish:** You can set and forget in the template if it's all the same. 

Makes the data more scannable because you only have to worry about the differences.

**Example**
```suggestion
```

Then set `text` in the test template because it'll load the template context and then what you've set.

```twig
{# test-alert-comparison.twig #}

{# Shared text for Alert and Site alert. #}
{% set text = "Lorem ipsum dolor sit amet sed do eiusmod." %}
```

**Output**
![image](https://github.com/user-attachments/assets/22e091ff-754c-4ba8-b5cb-9133e64b0f9a)

Reply:
Oooo...nice! Updated in [d42d4d3](https://github.com/uswds/uswds/pull/5636/commits/d42d4d390d28680add1c1a26c7e3c1982f68bec7)